### DISCUSSION TITLE
Issue: Pull Request Stuck in "Checking for ability to merge automatically" Status Hello GitHub Community,

### DISCUSSION THREAD

**Discussion by DevITSharpPRO** [community/community#130262] [Pull Requests] [answered]
### Select Topic Area

Bug

### Body

I am experiencing an issue with my pull request. Despite having all changes approved and receiving 1 approving review (1 approval), the status remains yellow with the message:

"Checking for ability to merge automatically… Hang in there while we check the branch’s status."

This state does not change, and the pull request seems to be stuck. I've tried waiting for a significant amount of time, but the status remains the same.

Has anyone else faced this issue, and are there any solutions or workarounds to resolve this?

Additional Information:

The branch has no conflicts.
All checks have passed.

Any help or insights would be greatly appreciated!

Thank you!

**Comment by SaintAngeLs** [comment, marked as answer]
Hello,

I've encountered this issue before and understand how frustrating it can be. Here are some steps you can try to resolve the problem:

- Ensure that all the required status checks are completed. Sometimes, additional checks (like CI/CD pipelines) might be running in the background. You can find this information in the "Checks" tab of your pull request.

- Sometimes, updating your branch with the latest changes from the base branch can help resolve the issue. You can do this via the command line or directly on GitHub by selecting "Update branch" if available.

For a more detailed discussion and additional troubleshooting steps, you might find this Stack Overflow post helpful: https://stackoverflow.com/questions/63315676/github-how-to-solve-hanging-pull-request.

If none of these steps work, consider reaching out to GitHub support for further assistance. They can provide more in-depth analysis and support for your specific situation.

I hope this helps!

**Reply by jonathans199** [reply]
yes, Github is having slowdown in PRs merging
![Screenshot 2024-06-28 at 2 49 52 PM](https://github.com/community/community/assets/30424887/0304d0c7-96cb-4505-b110-e4e6116cbdc6)

**Comment by ckybryan** [comment]
I think it's happening again, whie github status is still green 
<img width="1426" height="869" alt="image" src="https://github.com/user-attachments/assets/52222184-6ae1-4eeb-855a-82a702874762" />

**Reply by sambuysse** [reply]
Same thing happening here...

Edit: Seems to have resolved itself after 10 minutes.

**Reply by amddeus** [reply]
Also encountering the issue. Its been stuck over an hour

**Reply by chaojian-zhang** [reply]
Having same issue right now. [Status page](https://www.githubstatus.com/) (someone should really mention the link in this discussion😂) all green.

Update: Recovered after about 5 minutes.

**Comment by ppamorim** [comment]
Happened again, Github status page shows it's fully working.

<img width="663" height="301" alt="Screenshot 2025-11-11 at 16 20 40" src="https://github.com/user-attachments/assets/8de2fde6-f204-4ce1-8ebc-22c47d506238" />

What's the point to use those rules if we can't have a stable platform to use?

**Comment by realFranco-alenia** [comment]
This same issue happens to me, for the first time as well 🤔:

<img width="945" height="203" alt="Screenshot 2025-11-17 at 12 38 02" src="https://github.com/user-attachments/assets/dd99d2db-e143-43ac-b642-a7a92f86c3ef" />

Perhaps the GitHub Status website (at this point in time) is totally green. Even in the `eu` subdomain considering that currently I am in EU.

**Reply by BlueTogepi** [reply]
me too, really funny when I saw your comment as now XD

**Reply by Dmtreaqq** [reply]
lol same time

**Reply by BlueTogepi** [reply]
update: the issue's just gone after waiting for some time, not sure what's happening though

**Reply by realFranco-alenia** [reply]
## Solution

- Wait.
- If nothing happends, rewrite the last commit `git commit --amend` and push the change.
- Wait more.

<img width="951" height="177" alt="Screenshot 2025-11-17 at 12 47 31" src="https://github.com/user-attachments/assets/6b257e04-6d2f-442e-8763-8534de87b864" />

In my ciase I don't need to create branches.

**Comment by Dmtreaqq** [comment]
Today it happened to me for the first time. What I did. Closed pull request, and created a new one totally same and it works fine now.

**Reply by Alexander89** [reply]
same here. Status looks ok

**Comment by movsianikov** [comment]
experiencing same issue for about 20 minutes already. Tried solutions proposed here - nothing works.

**Reply by n-batalha** [reply]
getting this issue too now. this is on a repo without checks yet

edit: it finished the second I post this ofc. But it took 50min in total

**Reply by mpro7** [reply]
Facing the issue for 10 minutes do far, despite https://www.githubstatus.com/ showing all green.

EDIT: Worked right after posting this reply. Not sure if complaining here could be named a working solution, but coincidentally worked 😉

**Reply by danielatjumo** [reply]
same here cant merge PRs

**Reply by movsianikov** [reply]
finally went through after 30+ minutes of waiting.

**Reply by engelkes-finstreet** [reply]
Same here, cannot merge my PRs. Fingers crossed that posting here fixes the issue!

**Reply by denisblaga** [reply]
For me, the solution was to push a change, I added a comment, and then it turned green.

**Reply by hiUbaid** [reply]
faced the same problem, got fixed in like 10-15 minutes

**Reply by shovon-LifeArk** [reply]
facing same issue, stuck for about 15 minutes,

**Reply by bug-author** [reply]
Happening again

**Reply by matifrancois** [reply]
Same problem here, even with a very basic PR

**Reply by assofohdz** [reply]
Yep

**Reply by abhimanu-sharma** [reply]
Same problem here

**Reply by sarunassaulis** [reply]
Same here

**Reply by MoritzLost** [reply]
Same here

**Reply by dejan7** [reply]
same!

**Reply by MMMartt** [reply]
Same here

**Reply by leoalvarenga-fitec** [reply]
Same here. Still hanging, even with no additional checks to run...

It's been that way for a while now...

**Reply by mathewsunaal** [reply]
same

**Reply by gooooer** [reply]
I see same error

**Comment by hiUbaid** [comment]
Just blame it on github and move on

**Comment by rvesse** [comment]
One possible cause (**NB** this solution won't be relevant for some users) is if your repository has errors in its [`CODEOWNERS`](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) and your repository/organisation branch rulesets require approvals from code owners in order to merge.

To see if this affects your repository firstly navigate to the `.github/CODEOWNERS` file in your repository, if you see the following banner:

<img width="653" height="63" alt="Invalid CODEOWNERS banner" src="https://github.com/user-attachments/assets/81f7b43f-e975-412c-8e95-93f07c0daf30" />

Then your `CODEOWNERS` file is invalid and needs fixing, clicking on the `...` icon in the banner will pop up a modal describing what's actually wrong with the file e.g.

<img width="618" height="281" alt="Example CODEOWNERS errors" src="https://github.com/user-attachments/assets/e44aa0ea-e0e3-4aca-ab92-a8edf1b57323" />

One possible error that can occur if you've recently transferred a repository between organisations (or changed user permissions) is that all users in a `CODEOWNERS` **MUST** have write permissions for a repository.

In the above example the repository in question had recently been transferred between GitHub organisations and some users were previously inheriting write permissions from the old organisation **but** after transferring a repository they no longer had write permissions on it.  After granting the affected users write permissions on the repository my `CODEOWNERS` file is now again reported as valid:

<img width="251" height="64" alt="Valid CODEOWNERS banner" src="https://github.com/user-attachments/assets/9c27c87e-4f42-4d4b-a1e3-3f63aa2cf3de" />

**Reply by movsianikov** [reply]
I don't have this CODEOWNERS file.

**Reply by steven-radford** [reply]
i have the same probelm and i dont have a CODEOWNERS file

**Reply by rvesse** [reply]
This was just one solution I ran into for my version of the problem, I have updated my answer to make it clear that this won't be the cause of/solution to the problem for all users

**Comment by ua-knight** [comment]
This happens to me all the time. Lately about 90% of all merges. However, if I wait 5 seconds and reload the browser page, it always continues and completes the check.

**Reply by mikeKuester** [reply]
I have the same problem now. It starts one or two weeks ago. If I open the same url in a new browser it's ready! It seems to me, that only the website didn't get the update?

<img width="1586" height="363" alt="image" src="https://github.com/user-attachments/assets/d7149fce-f6c3-449d-8ab1-e9ab91ae01c6" />

But the two checks get their updates:
<img width="818" height="327" alt="image" src="https://github.com/user-attachments/assets/0f1afd79-5c34-4e42-89ad-45d070105ba4" />

**Reply by duncanplatt** [reply]
Experiencing this today.... Annoying.

**Comment by imnatgreen** [comment]
using the CLI worked for me (i.e. `gh pr merge`) whilst the browser UI was still stuck on checking

**Comment by PeterStub** [comment]
Not sure if its OK - but I just do a re-load...Then its all green and ready to be merged.

**Comment by Wlademirpx** [comment]
For me, what worked was simply making a new commit.
